docs/application/toc_all.md

```diff
@@ -793,6 +793,8 @@
 ###### [Overview](/application/native/guides/ui/dali/dali-rendering.md)
 ###### [Shape Drawing](/application/native/guides/ui/dali/rendering-tutorial.md)
 ###### [SVG Rendering](/application/native/guides/ui/dali/svg-rendering.md)
+##### Capture
+###### [Capture](/application/native/guides/ui/dali/capture.md)
```

Comment:
Question to this line.
Are you going to add another sub pages under 'Capture'?

If the capture.md is the only page under "Capture", 
We don't need to make tree as [this comment](https://github.com/Samsung/tizen-docs/pull/1017#discussion_r457866161).

>We don't make a tree when it has only one topic.

As like the "Resources" case on line 791, 
We can change line 796 and line 797 to below one line.

\##### \[Capture](/application/native/guides/ui/dali/capture.md)

Reply:
Thank you I fixed it.